src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> taskArr;
```

Comment:
I like how in the variable name you specified what the ArrayList is containing. It made me realise the areas of my code which I can improve on.